extern/sector-storage/selector_alloc.go

```diff
@@ -26,6 +26,8 @@ func newAllocSelector(index stores.SectorIndex, alloc storiface.SectorFileType,
 	}
 }
 
+// allocSelector only allows selecting a worker which supports the given tasktype and has enough space in the configured path type(`s.ptype`) in
+// the sector store to store a sector file of type/s `s.alloc` for a sector with the given size(`spt.SectorSize()`).
```

Comment:
This is too detailed / implementation specific for godocs. Private code references should be _inside_ the method body.

It also doesn't explain what allocSector actually does, it just expresses a constraint.